tools/swift-format/Sources/Rules/FullyIndirectEnum.swift

```diff
@@ -12,5 +13,67 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#enum-cases
 public final class FullyIndirectEnum: SyntaxFormatRule {
+  public override func visit(_ node: EnumDeclSyntax) -> DeclSyntax {
+    let enumMembers = node.members.members
+    
+    // Verifies all cases are indirect
+    var it = node.members.members.makeIterator()
```

Comment:
Can you split this into another method? Also, you should be able to just write `for caseMember in node.members.members { ... }`